File: platforms/kotlin/examples/scene/material/metallic_roughness_texture.kt
import org.fragmentcolor.*

val renderer = Renderer()
val mr_map_bytes = listOf(0.0f, 200.0f, 50.0f, 255.0f, 0.0f, 240.0f, 80.0f, 255.0f, 0.0f, 180.0f, 30.0f, 255.0f, 0.0f, 220.0f, 60.0f, 255.0f, .0f)
val mr_map = renderer.createTexture(TextureInputMobile.Bytes(mr_map_bytes.let { ba -> ByteArray(ba.size) { i -> ba[i].toInt().and(0xFF).toByte() } }), null)
val mat = Material.pbr().metallicRoughnessTexture(mr_map)

File: platforms/python/examples/scene/material/metallic_roughness_texture.py
from fragmentcolor import Material, Renderer

renderer = Renderer()
mr_map_bytes = [
    0, 200, 50, 255,
    0,   240, 80, 255,
    0,   180, 30, 255,
    0,   220, 60, 255,
]
mr_map = renderer.create_texture(mr_map_bytes, size=[2, 2])
mat = Material.pbr().metallic_roughness_texture(mr_map)